porting/index.md

```diff
@@ -50,6 +55,8 @@ repo init -u https://github.com/Halium/android -b halium-5.1
 
 #### Step 3: Download the source code
 
+*Note:* the repo tool takes some time to download the sources. You need a little patience here. Also it could make sense to first read the next part about the Android tree configuration, since there you need another repo sync call to download everything.
```

Comment:
What are you referring to with "another repo sync call"? I didn't do another one and there is only one mentioned in the guide.